### PR TITLE
Add Portuguese Hate Speech dataset

### PR DESCRIPTION
**Pull request by hugoabonizio**
Binary Portuguese Hate Speech dataset from [this paper](https://www.aclweb.org/anthology/W19-3510/).

### CODE REVIEW

File: datasets/hate_speech_portuguese/hate_speech_portuguese.py [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+    VERSION = datasets.Version("1.1.0")
+
+    def _info(self):
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=datasets.Features(
+                {
+                    "text": datasets.Value("string"),
+                    "label": datasets.ClassLabel(names=["hate", "no-hate"]),
```
Comment (lhoestq) [inline datasets/hate_speech_portuguese/hate_speech_portuguese.py:66]:
```suggestion
                    "label": datasets.ClassLabel(names=["no-hate", "hate"]),
```

Can you make this change and regenerate the dataset_infos.json file please ?

We usually set the positive label at index 1 in the list of label names (while here the positive label `hate` was at index 0)

**Review by lhoestq — COMMENTED** [review]
Nice thank you !
I just left one comment

**Comment by hugoabonizio** [comment]
@lhoestq done!

**Review by lhoestq — APPROVED** [review]
Awesome thanks :)

**Comment by lhoestq** [comment]
merging since the CI is fixed on master

**Pull request merged**